ui/lib/core/addon/templates/components/replication-action-promote.hbs

```diff
@@ -1,33 +1,33 @@
-<div class="action-block is-rounded" data-test-promote-replication>
-  <div class="action-block-info">
-    <h4 class="title is-5 is-marginless">
-      Promote cluster
-    </h4>
-    <p>
-      Promote this cluster to a {{replicationDisplayMode}} primary
-    </p>
-  </div>
+<div onkeyup={{action "onSubmit" "promote" model.replicationAttrs.modeForUrl (hash dr_operation_token=dr_operation_token primary_cluster_addr=primary_cluster_addr force=force)}}>
```

Comment:
wrapping div on all replicationAction components (but disable and demote) so that they trigger the check on escape.